src/selectors/entities/files.js

```diff
@@ -25,9 +26,13 @@ export function getFilePublicLink(state) {
 
 export function makeGetFilesForPost() {
     return createSelector(
-        [getAllFiles, getFilesIdsForPost, getCurrentUserLocale],
-        (allFiles, fileIdsForPost, locale) => {
-            const fileInfos = fileIdsForPost.map((id) => allFiles[id]).filter((id) => Boolean(id));
+        [getPost, getAllFiles, getFilesIdsForPost, getCurrentUserLocale],
+        (post, allFiles, fileIdsForPost, locale) => {
+            let fileInfos = post.metadata && post.metadata.files && [...post.metadata.files];
```

Comment:
Do we need this? I thought we are making changes to have files in the same key as before.

Reply:
0/5 on how to do it, this is definitely the easiest path

Reply:
This is definitely easy. I am confused on the approach we are talking with respect to the having files in state.

From last discussion offline we decided to have files in the same redux key as before so i am curios as to which direction we are going?

@hmhealey thoughts?